src/definitions/modules/accordion.less

```diff
@@ -176,6 +176,56 @@
   }
 }
 
+& when (@variationAccordionCompact) {
+  /*--------------
+       Compact
+  ---------------*/
+
+  /* Default Styling */
+  .ui.compact.accordion:not(.styled) .title:not(.ui) {

```

Comment:
Why is `:not(.ui)` necessary? We don't have a `ui title` or `ui content` selector? What am i missing?

Reply:
I was trying to wonder why too. Seems it's already set in initial declaration:
```css
/* Default Styling */
.ui.accordion .title:not(.ui) {
  padding: @titlePadding;
  font-family: @titleFont;
  font-size: @titleFontSize;
  color: @titleColor;
}

/* Default Styling */
.ui.accordion:not(.styled) .title ~ .content:not(.ui),
.ui.accordion:not(.styled) .accordion .title ~ .content:not(.ui) {
  margin: @contentMargin;
  padding: @contentPadding;
}
.ui.accordion:not(.styled) .title ~ .content:not(.ui):last-child {
  padding-bottom: 0;
}
```

Maybe an old remnant code from v1 🤔

Reply:
Can you try to remove that `:not(.ui)` from everywhere in accordion.less ? I searched the whole source , there definately isn't any `.ui.*.content` or title selector, so i strongly believe we can removed that completely.

Reply:
Done by `af3ed42`.

I also doubt that `.ui.accordion .accordion` is still used nowadays.